src/main/kotlin/three/consulting/epoc/service/TimesheetService.kt

```diff
@@ -14,6 +14,15 @@ private val logger = KotlinLogging.logger {}
 @Service
 class TimesheetService(private val timesheetRepository: TimesheetRepository) {
 
+    fun findTimesheetForProjectId(id: Long): List<TimesheetDTO>? {
+        logger.info { "Looking for timesheet with project_id: $id" }
+        val timesheet: List<Timesheet> = timesheetRepository.findAllByProjectId(id)
```

Comment:
About function/variable naming. You're handling a collection of timesheets, so naming should reflect that. Function could be `findTimesheetsForProjectId` (plural) same as the variable being `timesheets` as it's not a single object.